tsconfig.json

```diff
@@ -9,10 +9,10 @@
     "reactNamespace": "React",
     "module": "ESNext",
     "moduleResolution": "Node",
+    "sourceMap": true,
     "paths": {
-      "~/*": [
-        "/*"
-      ]
+      "~/*": ["/*"],
+      "@divriots/starter-react-sui/*": ["*"]
```

Comment:
`/*` ?

Reply:
Although I see some have it for ~, like furious:
https://github.com/divriots/starter-furious/blob/main/tsconfig.json#L10
?

Reply:
(but I admit even if just for consistency both should use either '*' or '/*' :grimacing: )

Reply:
in Furious I decided not to allow acces to internal packages and export everything via 1 entry point, therefore only `index.js`

the reasons is that tokens are not well structured and Microsoft itself doesn't now yet how to structure them and export everything in the same manner, so I'm just following their best practice

Reply:
> Although I see some have it for ~, like furious:
> https://github.com/divriots/starter-furious/blob/main/tsconfig.json#L10
> ?

I'm confused, Furious doesn't have `/*`, this is what it has

```json
    "paths": {
      "~/*": ["*"],
      "@divriots/starter-furious": ["index.ts"]
    }
```

Reply:
Sorry, I wasn't clear enough
I meant to say that I could change to "/*", but "*" seems to work as well as some starter kits are using it like Furious :-)
Anyway I tested and it does work :+1:

Reply:
> Sorry, I wasn't clear enough
> I meant to say that I could change to "/_", but "_" seems to work as well as some starter kits are using it like Furious :-)
> Anyway I tested and it does work 👍

it works because double slash is normalised to a single slash somewhere down the line, but it's no good IMO